ckks/bootstrapping/bootstrapper.go

```diff
@@ -28,10 +28,21 @@ type bootstrapperBase struct {
 	ctsMatrices advanced.EncodingMatrix
 
 	q0OverMessageRatio float64
+
+	swkDtS *rlwe.SwitchingKey
+	swkStD *rlwe.SwitchingKey
+}
+
+// Key is a type for a CKKS bootstrapping key, wich regroups the necessary public relinearization
+// and rotation keys (i.e., an EvaluationKey).
+type Key struct {
```

Comment:
For consistency, I would change this type slightly to disambiguate the term "Key", which could refer to a secret.
For example:

```suggestion
type EvaluationKey struct {
    *rlwe.EvaluationKey
     SwkDtS *rlwe.SwitchingKey
     SwkStD *rlwe.SwitchingKey
}
```

This is still coherent with the package name: `bootstrapping.EvaluationKey`.

Reply:
works for me, although `bootstrapping.Key` should already reflect the purpose of the key.

Reply:
But doesn't help figuring out that this key is not private. With the suggestion, we are explicit that this object is an evaluation key. We could add "Public" somewhere but we didn't for the rest of the library.